### DISCUSSION TITLE
Soru işaretli yere hangi şekil gelmeli?

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#13] [Zeka Sorusu]
![s005](https://github.com/sonsuzus/sonsuzus/assets/52765796/f67e6712-fb40-4774-8358-39a236fe62e2)

Soru işaretli yere yanındaki şekillerden hangisi gelmeli?

**Comment by HabilCcabbar** [comment]
Üçüncü şekil.  Çünkü ana şekil 45 derece dönünce köşede bulunan küçük şekil bir kez, ana şekil 90 derece döndüğünde ise köşede bulunan küçük şekil iki kez köşe değiştirmektedir. Aynı örüntü ikinci satırda da var. Ama üçüncü satırda ilk resimde neden üçgen yok ve sadece çizgi onu çözemedim.
